### PR TITLE
Developing uncontrast

### PR DESCRIPTION
**Pull request by oryrosin**

### CODE REVIEW

**Review by ChenPeleg — DISMISSED** [review]
Some comments and recommendations, 
`buildHoverText` is a  unclear what goes on there so make it more understandable

File: src/uncontrast/pages/TheoriesComparison/TheoriesComparison.jsx [hunk 1]
```diff
@@ -0,0 +1,280 @@
...
+                  ]}
+                  config={plotConfig}
+                  layout={{
+                    width: isMoblile ? screenWidth : screenWidth / 3,
+                    height: isMoblile ? screenWidth : screenWidth / 3,
+                    showlegend: false,
+                    annotations: [
+                      {
+                        text:
+                          breakLongLines(chart.series_name, 11) +
```
[the remark this answers is not in the document]
Comment (oryrosin) [reply src/uncontrast/pages/TheoriesComparison/TheoriesComparison.jsx]: gog knows i've tried

File: src/uncontrast/pages/WorldMap/WorldMap.jsx [hunk 2]
```diff
@@ -0,0 +1,320 @@
...
+                  className="text-lg w-[300px]"
+                  aria-label="theories"
+                  closeMenuOnSelect={true}
+                  isClearable
+                  isMulti={true}
+                  value={theoryFamilies}
+                  options={theories}
+                  placeholder="Theories"
+                  onChange={(e) =>
+                    buildUrlForMultiSelect(e, "theory", searchParams, navigate)
```
Comment (ChenPeleg) [inline src/uncontrast/pages/WorldMap/WorldMap.jsx:279]: Good that it's extracted to another function  👌

File: src/uncontrast/pages/ParametersDistributionPie/ParametersDistributionPie.jsx [hunk 3]
```diff
@@ -0,0 +1,281 @@
...
+    const color = seriesName.color;
+
+    setGraphData([
+      //inner pie
+      {
+        name: "drilled",
+        direction: "clockwise",
+        insidetextorientation: "horizontal",
+        values: [1],
+        labels: [rawTextToShow(secondaryData.series_name)],
```
Comment (ChenPeleg) [inline src/uncontrast/pages/ParametersDistributionPie/ParametersDistributionPie.jsx:142]: 👌 Good that its logic is extracted

**Review by ChenPeleg — COMMENTED** [review]
Looks good some small comments

File: src/Utils/HardCoded.jsx [hunk 4]
```diff
@@ -10,181 +10,189 @@ export const uploadPaperUsedHeight = footerHeight + navHeight + 20;
...
-        filename: `${window.location.href}`,
-        scale: 1, // Multiply title/legend/axis/canvas sizes by this factor
-    },
-    displaylogo: false,
+  modeBarButtonsToRemove: ["pan", "select", "lasso", "autoscale2d"],
+  scrollZoom: false,
+  displayModeBar: true,
+  toImageButtonOptions: {
+    format: "svg", // one of png, svg, jpeg, webp
+    filename: `${window.location.href}`,
```
[the remark this answers is not in the document]
Comment (oryrosin) [reply src/Utils/HardCoded.jsx:23]: it download the file with the name of the plotqgraph

**Comment by ChenPeleg** [comment]
Greate

**Review by ChenPeleg — APPROVED** [review]

**Pull request merged**